tools/shell/shell.cpp

```diff
@@ -4666,37 +4666,39 @@ void ShellState::ProcessDuckDBRC(const char *file) {
-                               "   -no-stdin            exit after processing options instead of reading stdin\n"
-                               "   -nullvalue TEXT      set text string for NULL values. Default ''\n"
-                               "   -quote               set output mode to 'quote'\n"
-                               "   -readonly            open the database read-only\n"
-                               "   -s COMMAND           run \"COMMAND\" and exit\n"
-                               "   -safe                enable safe-mode\n"
-                               "   -separator SEP       set output column separator. Default: '|'\n"
-                               "   -table               set output mode to 'table'\n"
-                               "   -unredacted          allow printing unredacted secrets\n"
-                               "   -unsigned            allow loading of unsigned extensions\n"
-                               "   -version             show DuckDB version\n";
+static const char zOptions[] =
+    "   -ascii               set output mode to 'ascii'\n"
+    "   -bail                stop after hitting an error\n"
+    "   -batch               force batch I/O\n"
+    "   -box                 set output mode to 'box'\n"
+    "   -column              set output mode to 'column'\n"
+    "   -cmd COMMAND         run \"COMMAND\" before reading stdin\n"
+    "   -c COMMAND           run \"COMMAND\" and exit\n"
+    "   -csv                 set output mode to 'csv'\n"
+    "   -echo                print commands before execution\n"
+    "   -f FILENAME          read/process named file and exit\n"
+    "   -init FILENAME       read/process named file\n"
+    "   -[no]header          turn headers on or off\n"
+    "   -help                show this message\n"
+    "   -html                set output mode to HTML\n"
+    "   -interactive         force interactive I/O\n"
+    "   -json                set output mode to 'json'\n"
+    "   -line                set output mode to 'line'\n"
+    "   -list                set output mode to 'list'\n"
+    "   -markdown            set output mode to 'markdown'\n"
+    "   -newline SEP         set output row separator. Default: '\\n'\n"
+    "   -no-stdin            exit after processing options instead of reading stdin\n"
+    "   -nullvalue TEXT      set text string for NULL values. Default ''\n"
+    "   -quote               set output mode to 'quote'\n"
+    "   -readonly            open the database read-only\n"
+    "   -s COMMAND           run \"COMMAND\" and exit\n"
+    "   -safe                enable safe-mode\n"
+    "   -separator SEP       set output column separator. Default: '|'\n"
+    "   -table               set output mode to 'table'\n"
+    "   -unredacted          allow printing unredacted secrets\n"
+    "   -ui                  launches a web interface using the ui extension (configurable with .ui_command)\n"
```

Comment:
```suggestion
    "   -ui                  launches a web interface using the ui extension (configurable with .ui_command)\n"
    "   -unredacted          allow printing unredacted secrets\n"
```
alphabetical ordered